### PR TITLE
IN profile detail

### PR DESCRIPTION
**Pull request by Tomology**
I have integrated the profile details page and the listings page. Listings now link to a user's profile. 
As we are using the user ID in the URL, I have modified the get profiles routes to work off of the user ID rather than the profile ID.

### CODE REVIEW

File: client/src/pages/Profile.js [hunk 1]
```diff
@@ -55,23 +57,55 @@ const useStyles = makeStyles((theme) => ({
...
+          <Paper className={classes.profileBreakpoints}>
+            <ProfileDetailCard profileDetails={profileDetails} />
+          </Paper>
+          {profileDetails.isSitter && user._id !== match.params.user_id && (
+            <Paper
+              className={
+                classes.requestBreakpoints + " " + classes.centerContent
+              }
+            >
+              <ProfileRequestForm />
```
Comment (dproc96) [inline client/src/pages/Profile.js]: You should be able to easily integrate bookings by passing the userId as a prop to the Requests form

**Review by dproc96 — COMMENTED** [review]
Everything you've done so far looks good, but I spotted one small thing you could add to this

File: client/src/components/profile/ProfileDetailCard.js [hunk 2]
```diff
@@ -20,21 +19,33 @@ const useStyles = makeStyles(() => ({
...
       <CardMedia
-        image={Background}
+        image={
+          profileDetails.headerPicture
+            ? profileDetails.headerPicture
+            : Background
+        }
         alt="bg"
-        style={{ width: "100%", height: "300px", objectFit: "cover" }}
+        style={{
```
Comment (dproc96) [inline client/src/components/profile/ProfileDetailCard.js]: Are these inline styles resolved in dev already?

File: client/src/pages/Profile.js (hunk 1 above)
Comment (Tomology) [reply client/src/pages/Profile.js]: Added.

File: client/src/components/profile/ProfileDetailCard.js (hunk 2 above)
Comment (Tomology) [reply client/src/components/profile/ProfileDetailCard.js]: They were but for some reason they no longer are. I have fixed it now.

**Review by dproc96 — APPROVED** [review]
This looks good to me, I'm gonna open a new ticket to figure out some oddness I'm seeing with the time zones though

**Pull request merged**